Squeezer/src/main/java/uk/org/ngo/squeezer/NowPlayingFragment.java

```diff
@@ -658,15 +660,22 @@ private void updateSongInfo(@NonNull PlayerState playerState) {
 
             if (mFullHeightLayout) {
                 artistText.setText(song.getArtist());
+                albumText.setText(song.getAlbumName());
                 totalTime.setText(Util.formatElapsedTime(song.getDuration()));
             }
+            else {

```

Comment:
Please can you put the `else` on the same line as the `}` (and in the other places you did this in this change). Thanks.